modules/Header/ui/Header.tsx

```diff
@@ -123,13 +127,14 @@ export function Header({
                         >
                             {newsroomName}
                         </h1>
-                        {newsroom.newsroom_logo && (
-                            <Image
-                                layout="fill"
-                                objectFit="contain"
-                                imageDetails={newsroom.newsroom_logo}
+                        {newsroomLogo && (
+                            <UploadcareImage
+                                style={{ objectFit: 'contain' }}
+                                src={newsroomLogo.cdnUrl}
                                 alt="" // This is a presentation image, the link has text inside <h1>, no need to have it twice. See [DEV-12311].
                                 className={styles.logo}
+                                width={320}
+                                height={56}
```

Comment:
@kudlajz I'm just starting a spike on looking at how we can better accommodate peoples different sized logos.

Looking at the current Bea, we set an equivalent `max-width` and `max-height` of these values. Does this component result in the same outcome? i.e. can I assume that this will behave similarly as current, for the purposes of thinking what I do next?

![CleanShot 2024-05-07 at 10 56 16@2x](https://github.com/prezly/theme-nextjs-bea/assets/858646/b5d03585-ccb3-4796-b329-2b045689f6c6)

Reply:
Yes, the dimensions are still the same, I've just set the width and height so the component fetches the image in 320px width (twice that for Retina) and not more.

Reply:
Cool -  thanks for confirming! Just checking that that width and height weren't now hardcoded as the width and height of the output or anything like that - I'll assume the CSS is as is current for this PR in terms of how we treat logos, so this is v much just changing how we fetch the logo more optimally 👍

Reply:
> so this is v much just changing how we fetch the logo more optimally

Yes 🙂